src/ontology/mondo-edit.obo

```diff
@@ -53127,10 +53125,8 @@ synonym: "skin metatypical basal cell carcinoma" EXACT [DOID:4281]
 synonym: "skin metatypical carcinoma" EXACT [DOID:4281]
 xref: DOID:4281 {source="MONDO:equivalentTo"}
 xref: ICDO:8095/3 {source="NCIT:C66903"}
-xref: MEDGEN:362831 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: NCIT:C66903 {source="MONDO:equivalentObsolete"}
 xref: SCTID:254702000 {source="MONDO:equivalentTo"}
-xref: UMLS:C1883040 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:362831"}
```

Comment:
Confirmed this UMLS concept was removed: C1883040 last appeared in the 2024AA version of the UMLS. It was mapped to a broader CUI: [C4721806](https://uts.nlm.nih.gov/uts/umls/concept/C4721806).